pallets/token-factory/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+authors = ["PureStake"]
+edition = "2018"
+name = "token-factory"
+version = "0.1.0"
+
+[dependencies]
```

Comment:
check if any unused (`pallet-ethereum`?) and split into `dev-dependencies` and `dependencies